README.md

```diff
@@ -65,43 +119,21 @@ when processed, produce Markdown content. Here's an example:
 }
 ```
 
-**Important**: when adding or editing return values, be sure to edit both files. Use the following
-links for the reply type. Note: do not use `@reply-type` specifiers; use only the Markdown link.
-
-**Important**: all links below are under construction.
-
-```md
-@simple-string-reply: [Simple string reply](https://valkey.io/topics/protocol#simple-strings)
-@simple-error-reply: [Simple error reply](https://valkey.io/topics/protocol#simple-errors)
-@integer-reply: [Integer reply](https://valkey.io/topics/protocol#integers)
-@bulk-string-reply: [Bulk string reply](https://valkey.io/topics/protocol#bulk-strings)
-@array-reply: [Array reply](https://valkey.io/topics/protocol#arrays)
-@nil-reply: [Nil reply](https://valkey.io/topics/protocol#bulk-strings)
-@null-reply: [Null reply](https://valkey.io/topics/protocol#nulls)
-@boolean-reply: [Boolean reply](https://valkey.io/topics/protocol#booleans)
-@double-reply: [Double reply](https://valkey.io/topics/protocol#doubles)
-@big-number-reply: [Big number reply](https://valkey.io/topics/protocol#big-numbers)
-@bulk-error-reply: [Bulk error reply](https://valkey.io/topics/protocol#bulk-errors)
-@verbatim-string-reply: [Verbatim string reply](https://valkey.io/topics/protocol#verbatim-strings)
-@map-reply: [Map reply](https://valkey.io/topics/protocol#maps)
-@set-reply: [Set reply](https://valkey.io/topics/protocol#sets)
-@push-reply: [Push reply](https://valkey.io/topics/protocol#pushes)
-```
-
-## Styling guidelines
+### Styling guidelines
 
 Please use the following formatting rules (aiming for smaller diffs that are easier to review):
 
-* No need for manual lines wrapping at any specific length,
-  doing so usually means that adding a word creates a cascade effect and changes other lines.
-* Please avoid writing lines that are too long,
-  this makes the diff harder to review when only one word is changed.
-* Start every sentence on a new line.
+* Please avoid writing lines that are too long.
+  That makes the diff harder to review when only one word is changed.
+* Single linebreaks are not significant in Markdown, so when editing an existing
+  sentence or paragraph, don't change the existing linebreaks. That just makes
+  reviewing harder.
 
+### Checking your work
 
-## Checking your work
+(This section may not be up do date.)
```

Comment:
It does still work, but it is generating a lot of warnings. I guess we should create an issue to update this to make sure spelling still works.

Reply:
With all these warnings, I'd say it doesn't work. It's not usable.

In past, it used to ignore text within backticks, but not anymore it seems.

Reply:
I actually tried this locally now. (Installing npm pulled down thousands of deps....)

It seems to ignore stuff within backticks, but in many cases command arguments are just bold and not within backticks. It seems possible to fix by wrapping these in backticks.

Reply:
OK, I still think we can follow up on this separately. It would be nice to have it working.

Reply:
Here: #93